hw3/02-charts.js

```diff
@@ -10,8 +36,22 @@ const backgroundColors = [
   'rgba(40, 159, 64, 0.8)',
   'rgba(210, 199, 199, 0.8)',
   'rgba(78, 52, 199, 0.8)',
+  ...moreColors,
 ];
 
+function getRandomColor() {
```

Comment:
Good idea bringing this in here. I just stuck with the lists provided.

Reply:
Thank you! I just used random colors as its easier to do then pick distinct colors. As I got more houses than the number of colors provided